content/en/docs/contribute/intermediate.md

```diff
@@ -536,24 +536,27 @@ Alternatively, you can install and use the `hugo` command on your development ma
 
 ## Triage and categorize issues
 
-In any given week, a specific docs approver volunteers to do initial
-[triage and review of pull requests](#review-pull-requests) and issues. To get
-on this list, attend the weekly SIG Docs meeting and volunteer. Even if you are
-not on the schedule for the current week, you can still review PRs.
-
 People in SIG Docs are responsible only for triaging and categorizing
 documentation issues. General website issues are also filed in the
 `kubernetes/website` repository.
 
 When you triage an issue, you:
 
-- Assess whether the issue has merit. Some issues can be closed quickly by
-  answering a question or pointing the reporter to a resource.
-- Ask the reporter for more information if the issue doesn't have enough
-  detail to be actionable or the template is not filled out adequately.
-- Add labels (sometimes called tags), projects, or milestones to the issue.
-  Projects and milestones are not heavily used by the SIG Docs team.
-- At your discretion, taking ownership of an issue and submitting a PR for it
+- Validate the issue
+    - Assess whether the issue has merit. Some issues can be closed quickly by
+      answering a question or pointing the reporter to a resource. See the
+      [Support requests or code bug reports](#support-requests-or-code-bug-reports) section for details.
+    - Add the `triage/needs-information` label if the issue doesn't have enough
```

Comment:
My rule of thumb is that if an issue goes stale and has this label, it's OK to close.

How about mentioning that / some variation on that in this page?

Reply:
I recommend telling the reader how to interact with Prow here. (ie, start a line with `/triage needs-information`)

Reply:
@sftim -  I thought about putting in something about Prow and even typed half a sentence... then retracted. Definitely worth revisiting!

Reply:
Added a separate Learn about Prow section at top of page